package.json

```diff
@@ -63,7 +63,7 @@
     "@angular/core": "^4.2.0",
     "@angular/forms": "^4.2.0",
     "@angular/http": "^4.2.0",
-    "@angular/material": "^2.0.0-beta.7",
+    "@angular/material": "^2.0.0-beta.12",
```

Comment:
We should lock this to version `2.0.0-beta.12` so it won't break if a new release is published with breaking changes.